lib/octopus/collection_association.rb

```diff
@@ -7,8 +7,7 @@ def self.included(base)
                              :load_target, :reload, :size, :select
       else
         base.sharded_methods :reader, :writer, :ids_reader, :ids_writer, :create, :create!,
-                             :build, :any?, :count, :empty?, :first, :include?, :last, :length,
-                             :load_target, :many?, :reload, :size, :select, :uniq
+			     :build, :empty?, :include?, :load_target, :reload, :size, :select
```

Comment:
whats up with the formatting here?

Reply:
oh weird, I made the change with vim so I must have done something weird